### ISSUE TITLE
[EN][Core] server info requires being alive

### ISSUE THREAD

**Issue by CrypticWizard** [Atlantiss/BugTracker#9460]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Cannot use .server info while dead.

**Current behaviour**: An error message is shown -- Cannot do this while dead, or words to that effect.

**Expected behaviour**: .server info should be available regardless of character state.

**Steps to reproduce the problem**:

1. Die.
2. Use .server info

**Revision**: 5482

**Comment by TranceAccount** [comment]
Did you try it with /g or /1 ?
You can't SAY something when you are dead, that is what causes this I think.

**Comment by CrypticWizard** [comment]
No, I did not try that. Then again, I wasn't looking for a fix for myself. It's a bug report. This is unexpected and confusing behaviour given the default nature of /say. Would be kind of silly if we had to end up putting the note: "Oh, and make sure you /w the .server info command to yourself" in the bug report template.

Just tried it and indeed it works with /g, even if you're not in a guild. Which is silly in and of itself but well, I suppose this bug report is inaccurate and invalid like this. Will close. Even though I still think it's confusing behaviour.
